Bridges/HttpKernel.php

```diff
@@ -240,12 +238,22 @@ protected function mapResponse(HttpResponse $reactResponse, SymfonyResponse $syR
 
         $reactResponse->writeHead($syResponse->getStatusCode(), $headers);
 
-        $stdOut = '';
-        while ($buffer = @ob_get_clean()) {
-            $stdOut .= $buffer;
+        // asynchronously get content
+        ob_start(function($buffer) use ($reactResponse) {
+            $reactResponse->write($buffer);
```

Comment:
@marcj 

> Since all stuff here is asycnhronouse we can not do async stuff here. That will break since reactResponse->end() is called immediately some lines below. Also I don't get the point of using async method here because between this line and $reactResponse->end is nothing that can do something async.

I'm not sure what you mean. We get multiple ob callbacks for a single `sendContent` so that is async in a manner. `$reactResponse->end` is only called after `sendContent` finishes (and `ob_end_flush` is called)- so all buffers have been cleared.

What could I do to resolve the issue you see? I have tried this ;)